hibernate-core/src/main/java/org/hibernate/engine/transaction/jta/platform/internal/JBossStandAloneJtaPlatform.java

```diff
@@ -48,6 +49,7 @@ protected TransactionManager locateTransactionManager() {
 	}
 
 	@Override
+	@SuppressWarnings("nullness")
```

Comment:
Use the helper class approach we talked about to reduce code duplication and handle nullness in that helper correctly.

Reply:
The main problem with this (for the CF) is the "invoke(null)" part; even if you check that the obtained Method reference is static (only case where you can pass in a null object reference), CF still complains.
So, I'm not seeing what you mean, basically.

Reply:
I suppose you will need a stub file entry for `Method.invoke()` then and annotate `@Nullable`.